content/en/serverless/installation/java.md

```diff
@@ -233,5 +296,4 @@ If you have trouble collecting monitoring data after following the instructions
 [11]: /tracing/connect_logs_and_traces/java/
 [12]: https://www.datadoghq.com/log4j-vulnerability/
 [13]: https://github.com/DataDog/datadog-lambda-java/releases
-[14]: https://docs.aws.amazon.com/lambda/latest/dg/invocation-layers.html
-[15]: /serverless/guide/troubleshoot_serverless_monitoring/
+[14]: /serverless/guide/troubleshoot_serverless_monitoring/
```

Comment:
Need to keep the deleted link because it's being used in "Install the Datadog tracing client" section.